lib/vagrant-cloudstack/config.rb

```diff
@@ -417,6 +422,9 @@ def finalize!
         # private ip is nil by default
         @private_ip_address     = nil if @private_ip_address == UNSET_VALUE
 
+        # expunge on destroy is nil by default
+        @expunge_on_destroy     = nil if @expunge_on_destroy == UNSET_VALUE
```

Comment:
Wouldn't is be better to make `@expunge_on_destroy = false` by default?
I understand that making it nil will have the same effect because later you do  `... if expunge_on_destroy != nil` (at L105 of terminate_instance https://github.com/schubergphilis/vagrant-cloudstack/pull/75/files#diff-b45563ef209cc71636c6684e13c4a82eR105) , but if you never set it to nil, you don't actually need the extra check, and can always pass the value to the API call.

This will probably have impact on the spec tests too.